src/pudl/metadata/fields.py

```diff
@@ -60,6 +60,150 @@
 # )
 
 FIELD_METADATA: dict[str, dict[str, Any]] = {
+    "delivered_gas_heat_content_mmbtu_per_mcf": {
+        "description": (
+            "The average annual heat content of gas delivered directly to consumers. "
+            "Note: operator 17617651KY has several values that are slightly higher than 1.2"
+        ),
+        "unit": "MMBtu_per_Mcf",
+        "type": "number",
+    },
+    "operational_consumption_facility_space_heat_mcf": {
+        "description": (
+            "Volume of natural gas consumed as fuel for space heat of the operator's own facilities within the report state. Reference conditions for measurement are 14.73 psia and 60° Fahrenheit."
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "operational_consumption_new_pipeline_fill_mcf": {
+        "description": (
+            "Volume of natural gas consumed as fuel for the operator's new pipeline fill within the report state. Reference conditions for measurement are 14.73 psia and 60° Fahrenheit."
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "operational_consumption_compressors_mcf": {
+        "description": (
+            "Volume of natural gas consumed as fuel for the operator's own pipeline distribution or storage compressor use within the report state. Reference conditions for measurement are 14.73 psia and 60° Fahrenheit."
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "operational_consumption_lng_vaporization_liquefaction_mcf": {
+        "description": (
+            "Volume of natural gas consumed by the operator for vaporization, liquefaction, and LNG fuel within the report state. Reference conditions for measurement are 14.73 psia and 60° Fahrenheit."
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "operational_consumption_vehicle_fuel_mcf": {
+        "description": (
+            "Volume of natural gas used in the operator's company-owned fleet within the report state. Reference conditions for measurement are 14.73 psia and 60° Fahrenheit."
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "operational_consumption_other_mcf": {
+        "description": (
+            "Volume of natural gas consumed as fuel by the operator for other purposes within the report state. Reference conditions for measurement are 14.73 psia and 60° Fahrenheit."
+        ),
+        "unit": "Mcf",
+        "type": "number",
+    },
+    "operational_consumption_other_detail": {
+        "description": (
+            "Free-text detail describing the operator’s specified “other purposes” for operational natural-gas consumption, corresponding to the volume reported in operational_consumption_other_mcf"
```

Comment:
Great description, thanks!

```suggestion
            "Free-text detail describing the operator’s specified “other purposes” for operational natural-gas consumption, corresponding to the volume reported in operational_consumption_other_mcf."
```